exasol_data_science_utils_python/schema/column_name_builder.py

```diff
@@ -1,17 +1,28 @@
-from typing import Union
+from typing import Optional
 
 from exasol_data_science_utils_python.schema.column_name import ColumnName
 from exasol_data_science_utils_python.schema.table_name import TableName
 
 
 class ColumnNameBuilder:
-    def __init__(self, column_name: Union[ColumnName, None] = None):
+    def __init__(self,
+                 name: Optional[str] = None,
+                 table_name: Optional[TableName] = None,
+                 column_name: Optional[ColumnName] = None):
+        """
+        Creates a builder for ViewName objects,
+        either by copying a ViewName object or
+        using the newly provided values.
```

Comment:
I believe the ViewName object is here parameter `column_name`. Maybe make this more explicit in the documentation.
Also, mention that parameter table_name has priority over parameter column_name in case both are not None

Reply:
copy paste error, good catch

Reply:
done